pkg/codegen/python/gen_test.go

```diff
@@ -75,6 +71,11 @@ func pyCompileCheck(t *testing.T, codeDir string) {
 	cmdOptions := integration.ProgramTestOptions{}
 	err = filepath.Walk(codeDir, func(path string, info filesystem.FileInfo, err error) error {
 		require.NoError(t, err) // an error in the walk
+
+		if info.Mode().IsDir() && info.Name() == "venv" {
```

Comment:
I noticed that it tries to recur into `venv` and spends a lot of time in there, short-circuit it.